aiida_widgets/structures.py

```diff
@@ -103,14 +107,8 @@ def _on_click_store(self, change):
         else:
             source_format = 'ASE'
 
-        # determine target format
-        if self.structure_node_class is None:
```

Comment:
I think I remember now why I added this `structure_node_class` variable: as a way for a user of the widget to get the node class without having to know about how it is implemented (data_format.value). E.g. you can just tab to autocomplete in ipython.

There is, however, a very simple way of combining your and my approach, which I think is better than either of them: use a property
```
@property
def node_class(self):
    return self.data_format.value
@node_class.setter
def node_class(self, value):
    self.data_format.value = value
```

Reply:
Done.